data/techs.dat

```diff
@@ -241,6 +241,16 @@ cost_list = 5000 | 45967 | 0
 pre_list = Heat Signature Reduction
 danger = 1
 
+[Metamaterial]
```

Comment:
Should be plural (Metamaterials); same with Advanced.  (Obviously this needs to change in all the places.)